File: PeachPie Extension Libraries/Peachpie.Community.ToolKit/src/Threading/Tasks/ManagedTaskEventArgs.cs
using System;

namespace Peachpie.Community.Threading.Tasks;

public class ManagedTaskEventArgs : EventArgs
{
    public int TaskId { get; }
    public object Result { get; }
    
    public ManagedTaskEventArgs(int taskId)
    {
        TaskId = taskId;
    }

    public ManagedTaskEventArgs(int taskId, object result)
    {
        TaskId = taskId;
        Result = result;
    }
}

File: PeachPie Extension Libraries/Peachpie.Community.ToolKit/.sdk/Peachpie/Community/Threading/Tasks/ManagedTaskEventArgs.php
<?php

namespace Peachpie\Community\Threading\Tasks;

use System\EventArgs;

class ManagedTaskEventArgs extends EventArgs
{
    public int $TaskId;
    public mixed $Result;

    /**
     * @param int $taskId
     * @param mixed|null $result
     */
    public function __construct(int $taskId , mixed $result = null)
    {
        $this->TaskId = $taskId;
        $this->Result = $result;
    }

}